djangochannelsrestframework/mixins.py

```diff
@@ -1,7 +1,56 @@
+from typing import Dict, Optional, OrderedDict, Union
+from rest_framework.utils.serializer_helpers import ReturnDict, ReturnList
+from django.db.models import Model, QuerySet 
 from rest_framework import status
 
 from .decorators import action
+from djangochannelsrestframework.settings import api_settings
+
+
+class PaginatedMixin:
```

Comment:
I was expecting `PaginatedMixin` to subclass `ListMixin` since it exposes `List` and thus remove custom `has attribute` logic in the `ListMixin`

Reply:
Ok will refactor it.